src/menu/menu.c

```diff
@@ -137,10 +137,11 @@ item_create(struct menu *menu, const char *text)
 	menuitem->selected.text = &menuitem->selected.buffer->scene_buffer->node;
 
 	/* Font buffers */
+	const char *arrow = show_arrow ? "›" : NULL;
```

Comment:
Does the arrow always fit into a single byte? I don't have much experience with UTF-8 const char assignments like this or wide chars / UTF storage in C in general.